### PR TITLE
Add NGLess

### PR DESCRIPTION
**Pull request by luispedro**
This is a tool for NGS with a focus on metagenomics:

http://ngless.embl.de/

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).

### CODE REVIEW

File: recipes/ngless/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - wget
+    - xz
+    - gcc # necessary for megahit
+  run:
+    - python
+    - bzip2 {{ CONDA_BZIP2 }}*
+    - gmp {{CONDA_GMP}}*
+    - zlib {{CONDA_ZLIB}}*
+    - libgcc # for gomp (megahit)
+    - xz
```
Comment (bgruening) [inline recipes/ngless/meta.yaml]: can we pin xz against: `xz 5.2.*`

Comment (luispedro) [reply recipes/ngless/meta.yaml]: Thanks. Fixed.

**Comment by luispedro** [comment]
Please don't merge. 

Given that I had to patch the `Makefile` for this recipe plus there was another bug report on building NGLess in static mode after the 0.5.0 release, I decided to just upstream those fixes and release 0.5.1 (later today, just need to make sure all the rebuilds are clean).

**Comment by luispedro** [comment]
Assuming the travis check finishes cleanly, I am going to assume that given @bgruening's LGTM, I can go ahead and merge.

**Comment by bgruening** [comment]
:+1:

**Pull request merged**